Algorithms.Tests/Encoders/HillEnconderTests.cs

```diff
@@ -15,7 +16,8 @@ public void DecodedStringIsTheSame()
             var encoder = new HillEncoder();
             var random = new Randomizer();
             var message = random.GetString();
-            var key = new double[,] { { 2, 4, 5 }, { 9, 2, 1 }, { 3, 17, 7 } };
+
+            var key = new double[,] { { 0, 4, 5 }, { 9, 2, -1 }, { 3, 17, 7 } };
```

Comment:
Why did you change the key, was it incorrect in some way?

Reply:
It increases the coverage of the test by including zeros and negative numbers. The alternative is to have multiple tests.